.gitignore

```
@@ -26,3 +26,10 @@
 /yarn-error.log
 yarn-debug.log*
 .yarn-integrity
+
+/app/assets/builds/*
+!/app/assets/builds/.keep
```

```suggestion
!/app/assets/builds/.keep
---
```

Let's include ".keep" because otherwise if the `builds` folder ever gets removed then the code will silently fail to load assets.